src/main/java/org/folio/search/model/index/InstanceSubResource.java

```diff
@@ -1,13 +1,14 @@
 package org.folio.search.model.index;
 
+import java.util.List;
 import lombok.Builder;
 import lombok.Data;
 
 @Data
 @Builder
 public class InstanceSubResource {
-  private String instanceId;
-  private String typeId;
   private String tenantId;
   private Boolean shared;
+  private int count;
+  private List<String> typeId;
```

Comment:
```suggestion
  private List<String> typeIds;
```

Reply:
It's typeId in the index, so to not break already existed query options it should stay `typeId`